planner/core/exhaust_physical_plans.go

```diff
@@ -199,7 +199,7 @@ func (p *LogicalJoin) getEnforcedMergeJoin(prop *property.PhysicalProperty) []Ph
 	// Check whether SMJ can satisfy the required property
 	offsets := make([]int, 0, len(p.LeftJoinKeys))
 	all, desc := prop.AllSameOrder()
-	if !all {
+	if !all || desc {
```

Comment:
Why ban the descent merge join plan? It seems to change it in the executor is not hard. Only keep similar with index merge join is ok.

Reply:
Perhaps we want a quick fix?

Reply:
We just release v3.0.9, and there is still time for fixing th executor before v3.0.10. Can we just fix the root cause?

Reply:
Fair enough, I'll fix it soon.